packages/docs/public/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
 
-    <title>Vuestic UI — Vue.js 3.0 UI Framework</title>
+    <title>Vuestic UI — Vue 3 UI framework</title>
```

Comment:
Font looks too big to me, but that's a matter of opinion.

Reply:
Fixed